.shopify-build/polaris-system-integration-test.yml

```diff
@@ -0,0 +1,29 @@
+containers:
+  default:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.18.0-yarn-1.21.1
+  styleguide:
+    docker: gcr.io/shopify-docker-images/apps/production/web-platform-ci-node:10.13.0-yarn-1.21.1
+
+shared:
+  build-polaris: &build-polaris
+    label: 'Build polaris'
+    container: default
+    timeout: 5m
+    dependencies:
+      - yarn
```

Comment:
Do [`- yarn: ~` ](https://github.com/Shopify/polaris-styleguide/blob/master/.shopify-build/polaris-styleguide.yml#L26) as a run step and you don't need the dependencies block